recipes/calliope/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "calliope" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "78c483e996d3f51f6285a505a82df002ac3079f33894563b2c5ec08dea265025" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<35]
+  entry_points:
+    - {{ name }} = {{ name }}.cli:cli
+
+requirements:
+  build:
+    - basemap
+    - bottleneck
+    - click >=3.3
+    - glpk ==4.59
+    - numexpr >=2.3.1
+    - numpy >=1.10.1
+    - pandas >=0.17.1,<0.18
+    - pyomo >=4.2,<4.3
```

Comment:
Recent versions of ``pyomo`` are available (https://github.com/conda-forge/pyomo-feedstock/) but my requirement was for an older version.

Failing to get that older version onto conda-forge, I'm updating my recipe PR with an as yet unreleased (on PyPI) version of my package which works with the most recent pyomo version. 

I'm versioning this ``0.4.0_rc1`` as I'd like to make sure the conda-forge package is up and running for the actual and soon-to-be-released 0.4.0, since the documentation will recommend using conda and conda-forge for installation...